clang/test/Modules/implicit-module-no-timestamp.cpp

```diff
@@ -0,0 +1,34 @@
+// UNSUPPORTED: system-windows
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: cd %t
+//
+// RUN: cp a1.h a.h
+// RUN: %clang -fmodules -fmodules-validate-input-files-content -Xclang -fno-pch-timestamp -fimplicit-modules -fmodule-map-file=module.modulemap -fsyntax-only  -fmodules-cache-path=%t test1.cpp
+// RUN: cp a2.h a.h
+// RUN: %clang -fmodules -fmodules-validate-input-files-content -Xclang -fno-pch-timestamp -fimplicit-modules -fmodule-map-file=module.modulemap -fsyntax-only  -fmodules-cache-path=%t test2.cpp
```

Comment:
Does this test need to use the clang driver for some reason, or can it be changed to use `%clang_cc1`, which is generally preferred for tests?

Reply:
> Does this test need to use the clang driver for some reason

Unfortunately, yes. The option `-fmodules-validate-input-files-content` is a driver only option.

Reply:
The cc1 version is `-fvalidate-ast-input-files-content`.

Reply:
Fixed, Thanks for the hint!